src/components/vm/vmMigrateDialog.jsx

```diff
@@ -0,0 +1,207 @@
+                   id="dest-uri"
+                   validated={validationFailed.destUri ? "error" : "default"}>
+            <TextInput id='dest-uri-input'
+                       validated={validationFailed.destUri ? "error" : "default"}
+                       helperTextInvalid={validationFailed.destUri}
+                       value={destUri}
+                       placeholder={cockpit.format(_("Example, $0"), "qemu+ssh://192.0.2.16/system")}
+                       onChange={setDestUri} />
+        </FormGroup>
+    );
+};
+
+const OptionsRow = ({ temporary, live, setTemporary, setLive, setStorage }) => {
+    return (
+        <FormGroup label={_("Options")}
+                   aria-label={_("Options")}
+                   id="options"
+                   hasNoPaddingTop>
+            <Checkbox id="live"
+                      isChecked={live}
+                      label={_("Run after migration")}
+                      onChange={b => {
+                          setLive(b);
+                          if (!b)
+                              setStorage("shared");
+                      }} />
+            {live && <Checkbox id="temporary"
+                               isChecked={temporary}
+                               label={
+                                   <>
+                                       {_("Move temporarily")}
+                                       <Tooltip
+                                           position="top"
+                                           content={
+                                               <>
+                                                   <p>{_("When shut off on the destination, this virtual machine will return to the original host in a shut off state.")}</p>
+                                                   <p>{_("This feature is often used for hardware maintanance or load-balacing.")}</p>
+                                               </>}>
+                                           <OutlinedQuestionCircleIcon id="migrate-tooltip" />
+                                       </Tooltip>
+                                   </>
+                               }
+                               onChange={setTemporary} />
+            }
+        </FormGroup>
+    );
+};
+
+const StorageRow = ({ storage, setStorage, live }) => {
+    const copyDisabled = !live;
+    let copyRadio = (
+        <Radio id="copy"
+               name="source"
+               label={_("Copy storage")}
+               isDisabled={copyDisabled}
+               isChecked={storage === "copy"}
+               onChange={() => setStorage("copy")} />
+    );
+
+    if (copyDisabled) {
+        copyRadio = (
+            <Tooltip id="storage-copy-tooltip"
+                     content={_("Offline migration doesn't allow copying storage")}>
+                {copyRadio}
+            </Tooltip>
+        );
+    }
+
+    return (
+        <FormGroup label={_("Storage")}
+                   aria-label={_("Storage")}
```

Comment:
You generally need aria-label only if you don't have a label. Did you copy that from somewhere? I will send up fixup and remove this, it does not hurt, it's just not necessary in the way this is written now.

Reply:
Oh sorry, i get why you did it, I now see that there are multiple fields inside.